src/components/AvatarList/AvatarList.tsx

```diff
@@ -40,8 +41,12 @@ export class AvatarList extends React.PureComponent<Props> {
   }
 
   getAvatars = () => {
-    return React.Children.toArray(this.props.children).filter(child =>
-      isComponentNamed(child, AVATAR_KEY)
+    return React.Children.toArray(this.props.children).filter(
+      child =>
+        isComponentNamed(child, AVATAR_KEY) ||
+        // This supports when Agent Avatars in HS App are wrapped
+        // with a Tooltip when they agent is typing.
+        isComponentNamed(child, TOOLTIP_KEY)
```

Comment:
FYI, this change will be remove in v3 because we don't filter any component anymore (putting more trust in dev to pass the right component needed as children)

```javascript
getAvatars = () => {
    return React.Children.toArray(this.props.children)
  }
```

Reply:
Cool. Good to know. Can I leave it or should I remove this?

Reply:
if you need it somewhere keep it, I'll make sure to remove it next time I merge master